src/mono/wasm/debugger/DebuggerTestSuite/TestHarnessStartup.cs

```diff
@@ -178,7 +178,7 @@ public void Configure(IApplicationBuilder app, IOptionsMonitor<TestHarnessOption
                         var client = new HttpClient();
                         var psi = new ProcessStartInfo();
 
-                        psi.Arguments = $"--headless --disable-gpu --lang=en-US --incognito --remote-debugging-port={devToolsUrl.Port} http://{TestHarnessProxy.Endpoint.Authority}/{options.PagePath}";
+                        psi.Arguments = $"--disable-gpu --lang=en-US --incognito --remote-debugging-port={devToolsUrl.Port} http://{TestHarnessProxy.Endpoint.Authority}/{options.PagePath}";
```

Comment:
this is painful to even test against, it could be a prop/option

Reply:
yeah, we could do that.